tests/fba.py

```diff
@@ -40,6 +40,26 @@ def test_fba_sbml(self):
 
         self.assertTrue(abs(res[1] - 0.8739215067486387) < 1e-03)
 
+    def test_modify_medium(self):
```

Comment:
Does this test that the medium are extracted correctly?

Reply:
Do you mean something like that? 
The check for the number of exchange reactions is in case in the future we enable to add an exchange reaction, rather than just editing its boundaries.

Reply:
Yes, exactly. Thanks!